apps/patient/src/views/Main.tsx

```diff
@@ -180,6 +179,8 @@ export const Main = () => {
     logo
   };
 
+  console.log(flattenedFills);
```

Comment:
Make sure we remove this before we merge

Reply:
removed